programs/programs.json

```diff
@@ -350,5 +350,17 @@
     "categories": [
       "NFT"
     ]
+  },
+    {
+    "name": "soundwork-contracts",
+    "description": "A decentralized Sound Network that seeks to revolutionize music production and redefine the music industry’s supply chain.",
+    "icon": "https://avatars.githubusercontent.com/u/132207133",
+    "repo": "https://github.com/SoundWorkLabs/marketplace-contracts",
+    "framework": "Anchor",
+    "categories": [
+      "NFT",
+      "Token",
+      "SPL"
```

Comment:
The "SPL" category is intended to use for only the programs in [solana-labs/solana-program-library](https://github.com/solana-labs/solana-program-library).
```suggestion
```